content/2022/videookulografija.md

```diff
@@ -1,4 +1,294 @@
+
+
+Dobijeni podaci imaju šum (kao što je treptaj, nenamerno pomeranje oka ili kamere, promena jačine osvetljenja i slično) koji se otklanjaju mean avarage filterom (filter srednje aritmetičke vrednosti). 
+Prolazići kroz bazu izračunava se aritmetička vrednost prethodnih n merenja koordinata zenice i te nove vrednosti se postavlja kao nove koordinate zenice. 
+Na taj način se dobija filtrirano merenje.
+
+Na slikama ispod se mogu videti grafovi filtriranih merenja, x i z ose u odnosu na vreme.
+
+
+![Koordinate x-ose u odnosu na t](/images/2022/videookulografija/filtriranx.jpg)(slika 11)
+
+
+![Koordinate y-ose u odnosu na t](/images/2022/videookulografija/filtrirany.jpg)(slika 12)
+
+### **3.2.3. Kalibracija**
+
+#### 3.2.3.1. Linearna regresija
+
+
+//dodati zasto sistem nije linearan
+
+Kako bi se uz pomoć detekcije zenice kontrolisao neki objekat na računaru potrebno je znati na osnovu koordinata centra zenice u koju tačku na ekranu je oko upereno. 
+Za to je korišćena linearna regresija, tj. polinomialna regresija jer sistem nije linearan. 
+
+Linearna regresija predstavlja modelovanje relacija između jedne ili više zavisnih promenljivih i jedne ili više nezavisnih promenljivih. 
+Model linearne regresije ima oblik $Y = X * b $, gde je Y vektor izmerenih vrednosti(u našem slučaju koordinate našeg pogkleda na ekranu) odnosno zavisna promenljiva, X vektor inputne promenljive(u našem slučaju koordinate zenice oka) odnosno nezavisna promenljiva  i b vektor koificijenta modela. 
+Vektor koificijenta modela b  
+
+
+#### 3.2.3.2. Polinomialna regresija
+Poliomilna regresija je specijalan slučaj višestruke linearne regresije gde se modeluje veza između zavisne i nezavisne promenljive, u ovom slučaju, između stvarnog položaja zenice i položaja tačke na ekranu, kao polinom n-tog stepena. 
+U našem slučaju, nema linearne korelacije između promenljivih (slika 1.1), pa se radi veće preciznosti promenjuje poliomilna regresija (slika 1.2).
+
+slika 1 slika 2
+
+Formula glasi: $y= b_0+b_1 \cdot x_1+ b_2 \cdot x_2+ b_2  \cdot x_2+...... b_n \cdot x_n$, gde su:
+b - koificijenti regresije
+n - stepen regresije (u našem slučaju $n = 3$)
+
+
+## **4. Rezultati i istraživanja**
+
+
+Model dobijen poliomilnom regresijom je proveren tako što je eksperiment snimiljen još jednom, ali ovog puta je dobijen predviđen položaja zenice.
+Na kraju, kao rezultat je dobijena tabela u kojoj, u odnosu na tačku na ekranu (x, y), se može videti pomerenost estimacije i standardna devijacija rasipanja (σx, σy).
+Rezultati sa jednim snimanjem imaja prosečan pomeraj 121.6 piksela i prosečne standardne devijacije raspipanja su σx = 43.48 i σy = 43.13. 
+Sa ovakvim rezultatima je veoma teško kontrolisati kursor ili igrati igricu. 
+Sa više snimanja i u drugačijim uslovima, kao npr. snimanje očiju svetlijih nijansi ili korišćenjem filtera za IR spektar greška bi se smanjivala i tako povećala mogućnost kontrolisanja kursora ili igranja igrica.
```

Comment:
Ne razumem ovo. Što bi se smanjivala greška povećavanjem merenja? Ja samo mislim da bi sa više merenja imali realniju sliku o sistemu